libs/perun/services/src/lib/auth.service.ts

```diff
@@ -206,7 +256,7 @@ export class AuthService {
         .loadDiscoveryDocument()
         .then(() => this.oauthService.refreshToken())
         .then(() => Promise.resolve())
-        .catch((err) => err);
+        .catch(() => Promise.resolve());
```

Comment:
The error will be handled in `ApiInterceptor` and since I believe we don't react to it in any specific way here, there is no need to return in. Instead empty `void` promise is returned.